bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -1,3 +1,5 @@
+
+---
```

Comment:
I just tried running make bundle. These two lines don't exist here. Yes they exists in config/crd/bases/oadp.openshift.io_dataprotectionapplications.yaml but it doesn't belong in bundle/manifests.

Just let `make bundle` do its thing. Hopefully that fixes it?
```suggestion
```

Reply:
Let me try that. Thanks @kaovilai

Reply:
updated